recipes/jp-aisi-ch.json

```diff
@@ -0,0 +1,37 @@
+{
+    "name": "Japan AISI Dataset Chinese",
+    "description": "",
```

Comment:
is there some description for this recipe?

Reply:
I will add it.